sites/public/pages/listings.tsx

```diff
@@ -156,7 +156,9 @@ const ListingsPage = () => {
       </div>
       {!listingsLoading && (
         <div>
-          {listingsData && <ListingsList listings={listingsData.items} />}
+          {listingsData && (
+            <ListingsList listings={listingsData.items} hideApplicationStatus={true} />
```

Comment:
I think you can just say `hideApplicationStatus` and it will be passed in as true. Do you also need the parens around `ListingList`?